conda_forge_tick/migrators_types.pyi

```diff
@@ -58,7 +58,7 @@ class MetaYamlOutputs(TypedDict, total=False):
     # TODO: Not entirely sure this is right
     build: BuildRunExportsDict
 
-class MetaYamlTypedDict(TypedDict, total=False):
+class RecipeTypedDict(TypedDict, total=False):
```

Comment:
Why was this type renamed?

Reply:
In my mind, both the parsed `recipe.yaml` and `meta.yaml` would return a dictionary that conforms to the same interface. For that, the name `MetaYamlTypedDict` doesn't make too much sense. 
Or do you think that is a bad idea?

Reply:
Yeah totally fine with the change. I just didn't understand the rationale.